ports/yolkheroes/README.md

```diff
@@ -0,0 +1,22 @@
+## Notes
+
+Please support the developers at https://store.steampowered.com/app/2417850/?snr=1_5_9__205 or https://pixelwestern.itch.io/yolk-hero. Purchasing the game is mandatory to run it.
+
+*Someone has to protect this realm... is it you? Become a guardian spirit and raise your very own Hero to stand against the Dark Lord.*
+
+Yolk Heroes is a Tamagotchi-style idle game where the player raises an Elf to become a hero. Think Digimon Tamagotchi meets Princess Maker as an idle game.
+
+The Gameboy-style art and screen scaling make it a good candidate for a portable platform.
+
+## Controls
+
+| Button | Action |
```

Comment:
Buttons should tell the user what they do, not tell what keyboard key it maps to.